### PR TITLE
Allow CACHET_AUTH_TOKEN as an env var.

### PR DESCRIPTION
**Pull request by michaelkaye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ environment variable. Please note that this is not supposed to end with a slash.
 The daemon is listening for http requests on port 8888.
 You should use a reverse proxy for TLS termination in front of that.
 
+If the `CACHET_AUTH_TOKEN` env var is set, then missing Authentication headers will use that value.
+Only set `CACHET_AUTH_TOKEN` if your are running in a secure environment.
```
Comment (jcgruenhage) [inline README.md]: I think we should make more clear what this means.

**Review by jcgruenhage — APPROVED** [review]
Even though the docs are not perfect, functionality wise this LGTM.

File: README.md (hunk 1 above)
Comment (michaelkaye) [reply README.md]: Been a bit more explicit on the failure and how to adjust for it.

Comment (jcgruenhage) [reply README.md]: thanks, this is definitely good to merge now.

**Pull request merged**